drake_cmake_installed/README.md

```diff
@@ -2,24 +2,50 @@
 
 This uses the CMake `find_package(drake)` mechanism to find an installed instance of Drake.
 
-# Locally on Xenial
+# Instructions
 
-Download the binaries and install into /opt. This will ensure that CMake is able
-to auto-discover the `drake-config.cmake` module. 
+These instructions are only supported for Ubuntu 16.04 (Xenial).
 
 ```
-# A nightly build
+# Install Prerequisites
+<TODO: install pre-requisite debs list>
+
+# Install Drake to /opt/drake
 # curl -O https://s3.amazonaws.com/drake-packages/drake/nightly/drake-20170719-xenial.tar.gz
 # Alternatively, the latest (usually last night's build)
 curl -O https://s3.amazonaws.com/drake-packages/drake/nightly/drake-latest-xenial.tar.gz
 sudo tar -xvzf drake-latest-xenial.tar.gz -C /opt
-mkdir build
-cd build
+
+# Build Everything
+mkdir build && cd build
 cmake ..
 make
+
+# Run a demo
+/opt/drake/bin/drake_visualizer &
+(cd src/particles && exec ./uniformly_accelerated_particle_demo)         
+
+# (Optionally) Install GTest
+# You could also explicitly pull gtest into the CMake build directly:
+#     https://github.com/google/googletest/tree/master/googletest
+sudo apt-get install libgtest-dev
+cd /usr/src/gtest
```

Comment:
BTW I think this line is no longer required / useful?

Reply:
yes, indeed, thanks.